### DISCUSSION TITLE
read-only consume directory

### DISCUSSION THREAD

**Discussion by alex-phillips** [jonaswinkler/paperless-ng#552] [Feature Requests]
I'd like to know if it's possible / can support be added for a read-only consume directory. I haven't tested this if it currently works, but mounting a consume directory as read-only in docker outputs some warnings that the directory is not writable.

The idea behind this is I currently store my documents in Nextcloud but would love to utilize paperless-ng for OCR and searching. What I would like to be able to do is mount my Nextcloud documents directory as read-only so the documents can stay organized via Nextcloud but have paperless-ng automatically ingest new document added to Nextcloud.

**Comment by jonaswinkler** [comment]
No, that's not possible. The entire idea of the consumption directory is that paperless moves files from there into it's media directory. When you also use `PAPERLESS_FILENAME_FORMAT`, you can have that media directory organized in any way you like.

I'd consider giving nextcloud read-only (!) access to the media directory instead. That way, you can use paperless to organize your documents, and still have access to them through nextcloud.

**Comment by LennartHallenberger** [comment]
I would also like the option to mount my documents folder into paperless as read-only.
It's important to me, that neither my folder nor my file names / structures are modified by paperless.
I don't even care if paperless stores a copy of each file somewhere else.
This would allow users to be more flexible combining paperless with other software (nextcloud, syncthing, rsync, etc.) depending on their document workflow.
